runscanner-illumina/README.md

```diff
@@ -0,0 +1,17 @@
+# Illumina Support for Run Scanner
+To use the Illumina support in Run Scanner, this additional package must be built.
+
+You will need a copy of [jsoncpp](https://github.com/open-source-parsers/jsoncpp) and an AutoTools-capable build environment. On Debian/Ubuntu:
```

Comment:
Did you look into Maven plugins to automate this? There's Native Maven Plugin that might do the trick: http://www.mojohaus.org/maven-native/native-maven-plugin/